PINCache/PINDiskCache.h

```diff
@@ -1,5 +1,6 @@
 //  PINCache is a modified version of TMCache
 //  Modifications by Garrett Moon
+//  Modifications by Sebastien Villar, Microsoft
```

Comment:
Do you mind taking this out? If it feels better to you, you can take out the Modifications by Garrett Moon too. I put that there when I forked the project originally because I modified nearly 100% of the project (that was my justification). But I'd rather not see this turn into a list of contributors. It would get, I hope, very long :)

You can take solace that you're name will show up in contributors and in the git blame :P